tests/k8s_tool.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
Please add this file to k8s directory

Reply:
we want to use it in our tests eventually (pod restart report)

Reply:
Agree. (I'm working on it)
But it is also good to have such tool independently.

Reply:
this is why I left it here I hope we can use it as a tool and as a library and integrate more tools into it. I've started laying the infrastructure. it's really basic right now but I hope it will grow better.